client/src/main/kotlin/no/nav/poao_tilgang/client/PolicyInput.kt

```diff
@@ -14,6 +14,11 @@ data class NavAnsattTilgangTilEksternBrukerPolicyInput(
 	val norskIdent: String
 ) : PolicyInput()
 
+data class NavAnsattNavIdentSkrivetilgangTilEksternBrukerPolicyInput(
+	val navIdent: String,
```

Comment:
burde vi bruke sterkere typer på clienten?

Reply:
Ja, jeg var litt usikker på det - den lå inne fra før som String, så derfor jeg har gjenbrukt det. Men mulig vi burde ha en egen type for det. Men feks norskIdent er jo Fnr og den er også String - så jeg er litt usikker på hvorfor de ikke er typet opp

Reply:
Jeg lager en oppgave på dette i Trello som vi kan se på senere.